frontend/src/sub-components/RoundLights.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+const Dot = styled.div`
+  display: flex;
+  height: 6rem;
+  width: 6rem;
+  border-radius: 50%;
+  margin-right: 10%;
+`;
+
+class RoundLights extends React.Component {
+  constructor(props) {
+    super(props);
+    this.statusColors = [];
+    /*
+     * Each rounds color status is appended to the status color array,
+     * where green represents completed and gray represents rounds
+     * that have not yet been completed.
+     */
+    for (let i = 0; i < this.props.currentRound; i++) {
+      this.statusColors.push("#1aa260");
```

Comment:
statusColors never changes, so you can make this once and reuse it. Then this component could be a function component.

Reply:
done.